Add details of shopping_list_updated events (#24080)
home-assistant/home-assistant.github.io: source/_integrations/shopping_list.markdown

| Data payload attribute | Description                                                                                                        |
|------------------------|--------------------------------------------------------------------------------------------------------------------|
| `action`               | What action was taken on the item. Either `add` for a new item being added, or `update` for an item being updated. |
| `item`                 | A dictionary containing details of the item that was updated.                                                      |
| `item.id`              | A unique ID for this item                                                                                          |
| `item.name`            | The text attached to the item, for example `Milk`                                                                  |
| `item.complete`        | A boolean indicated whether the item has been marked as complete.                                                  |

{% raw %}

```yaml
alias: "Notify on new shopping list item"
trigger:
  - platform: event
    event_type: shopping_list_updated
    event_data:
      action: "add"

> condition: []

Reviewer: Unneeded condition statement

```suggestion
```

## Using in Automations

> A `shopping_list_updated` event is triggered when items in the list are modified, with the following data payload attached to it. This can be used to trigger automations such as sending a push notification when someone adds an item to the shopping list, which when clicked will open the list.

Reviewer: ```suggestion
A `shopping_list_updated` event is triggered when items in the list are modified, with the following data payload attached to it. This can be used to trigger automations such as sending a push notification when someone adds an item to the shopping list, which when clicked, will open the list.
```

| Data payload attribute | Description                                                                                                        |
|------------------------|--------------------------------------------------------------------------------------------------------------------|
| `action`               | What action was taken on the item. Either `add` for a new item being added, or `update` for an item being updated. |
| `item`                 | A dictionary containing details of the item that was updated.                                                      |
| `item.id`              | A unique ID for this item                                                                                          |
| `item.name`            | The text attached to the item, for example `Milk`                                                                  |
| `item.complete`        | A boolean indicated whether the item has been marked as complete.                                                  |

{% raw %}

```yaml

> alias: Notify on new shopping list item

Reviewer: ```suggestion
alias: "Notify on new shopping list item"
```

trigger:
  - platform: event
    event_type: shopping_list_updated
    event_data:

>       action: add

Reviewer: ```suggestion
      action: "add"
```

condition: []
action:
  - service: notify.notify
    data:

>       message: '{{ trigger.event.data.item.name }} has been added to the shopping list'

Reviewer: ```suggestion
      message: "{{ trigger.event.data.item.name }} has been added to the shopping list"
```

      data:
        clickAction: "/shopping-list"
        url: "/shopping-list"

> mode: single

Reviewer: ```suggestion
```

```

{% endraw %}

> 

Reviewer: ```suggestion
```
